Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/energy.yml

```
@@ -44,7 +46,7 @@
     impactEffect: BulletImpactEffectRedDisabler
     damage:
       types:
-        Heat: 15 
+        Heat: 17 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)